src/sentry/integrations/github/client.py

```diff
@@ -656,8 +656,6 @@ def get_blame_for_file(
             if contents.get("data", {}).get("repository", {}).get("ref", {}) is None:
                 raise ApiError("Branch does not exist in GitHub.")
 
-            sentry_sdk.capture_exception(e)
```

Comment:
@NisanthanNanthakumar doesn't this capture other errors that might be valid? Aka not just branch not valid?